abcd/backends/atoms_opensearch.py

```diff
@@ -174,13 +174,14 @@ def from_atoms(
         return obj
 
     @property
-    def _id(self):
+    def _id(self) -> Union[str, None]:
```

Comment:
```suggestion
    def _id(self) -> Optional[str]:
```

Reply:
I considered this, but does it make sense to use optional for returns?

Reply:
I would, I see it as the `Maybe` from many functional languages such as Haskell where it means `Just [ans]` or `Nothing`